plutus-core/cost-model/data/models.R

```diff
@@ -12,6 +12,16 @@ library(broom,   quietly=TRUE, warn.conflicts=FALSE)
 ## require changes here to get sensible results.
 
 
+## At present, times in the becnhmarking data are typically of the order of
+## 10^(-6) seconds. We scale these up to milliseconds because the resulting
+## numbers are much easier to work with interactively.  For use in the Plutus
+## Core cost model we scale times up by a further factor of 10^6 (to
```

Comment:
So why not just go to picoseconds here?

Reply:
I think it says that somewhere.  The benchmark times are all like 10^-6 or 10^-7 seconds, and I quite often work with them interactively.  If you look at the raw times everything looks like  this:

```
3183         GtByteString   8192   8192 3.481928e-06 3.469370e-06 3.538704e-06 7.348516e-08 1.702707e-08 1.636750e-07
3184         GtByteString  16384  16384 5.324943e-06 5.260484e-06 5.560930e-06 3.884277e-07 3.383087e-08 8.232254e-07
3185         GtByteString  32768  32768 1.003767e-05 9.985790e-06 1.011762e-05 2.068979e-07 1.177017e-07 3.417348e-07
3186         GtByteString  65536  65536 1.803395e-05 1.802233e-05 1.807126e-05 6.594140e-08 2.184261e-08 1.332923e-07
3187         GtByteString 131072 131072 3.401697e-05 3.387513e-05 3.454369e-05 9.234961e-07 6.519390e-07 1.602633e-06
3188      VerifySignature      0     -1 7.272717e-06 7.196651e-06 7.451649e-06 3.881881e-07 2.763596e-07 5.720442e-07
3189      VerifySignature      0     -1 7.232630e-06 7.150771e-06 7.395957e-06 3.957633e-07 1.843607e-07 6.171057e-07
3190      VerifySignature      1     -1 7.160197e-06 7.150143e-06 7.170202e-06 3.222451e-08 2.571712e-08 4.102232e-08
3191      VerifySignature      2     -1 7.331839e-06 7.223802e-06 7.499512e-06 4.563588e-07 2.533636e-07 6.559043e-07
3192      VerifySignature      4     -1 7.176682e-06 7.136230e-06 7.342736e-06 2.452496e-07 3.308412e-08 5.171120e-07
```
but if you go to picoseconds it's
```
3183         GtByteString   8192   8192     3481928     3469370     3538704 7.348516e+04    17027.069 1.636750e+05
3184         GtByteString  16384  16384     5324943     5260484     5560930 3.884277e+05    33830.873 8.232254e+05
3185         GtByteString  32768  32768    10037668     9985790    10117624 2.068979e+05   117701.664 3.417348e+05
3186         GtByteString  65536  65536    18033954    18022329    18071260 6.594140e+04    21842.605 1.332923e+05
3187         GtByteString 131072 131072    34016965    33875129    34543686 9.234961e+05   651938.953 1.602633e+06
3188      VerifySignature      0     -1     7272717     7196651     7451649 3.881881e+05   276359.603 5.720442e+05
3189      VerifySignature      0     -1     7232630     7150771     7395957 3.957633e+05   184360.653 6.171057e+05
3190      VerifySignature      1     -1     7160197     7150143     7170202 3.222451e+04    25717.124 4.102232e+04
3191      VerifySignature      2     -1     7331839     7223802     7499512 4.563588e+05   253363.619 6.559043e+05
3192      VerifySignature      4     -1     7176682     7136230     7342736 2.452496e+05    33084.118 5.171120e+05
```
Neither of those is very easy to work with.

I think this is OK because the milliseconds data never really escapes from R.